### PR TITLE
Fixed deployment/CI versions

### PR DESCRIPTION
**Pull request by justinboswell**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,7 +7,8 @@ on:
       - '!master'
 
 env:
-  BUILDER_VERSION: v0.5.3
+  BUILDER_VERSION: more-fixes
+  BUILDER_SOURCE: channels
```
Comment (graebm) [inline .github/workflows/ci.yml]: does this need to change before merge?

File: .builder/jdk.py [hunk 2]
```diff
@@ -49,8 +49,10 @@ def install(self, env):
 
         sh = env.shell
 
+        target = '{}-{}'.format(env.spec.target, env.spec.arch)
+
         # If this is a local build, check the local machine
-        if not env.toolchain.cross_compile:
+        if not env.toolchain.cross_compile or target not in URLs:
```
Comment (graebm) [inline .builder/jdk.py]: do you still need jdk.py here? didn't you just move it into aws-crt-builder?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (justinboswell) [reply .github/workflows/ci.yml]: yes

File: .builder/jdk.py (hunk 2 above)
Comment (justinboswell) [reply .builder/jdk.py]: yes, it should have been deleted, will fix

**Pull request merged**